src/go/rpk/pkg/tuners/redpanda_checkers.go

```diff
@@ -71,8 +73,17 @@ func NewConfigChecker(conf *config.Config) Checker {
 		Fatal,
 		true,
 		func() (interface{}, error) {
-			ok, _ := conf.Check()
-			return ok, nil
+			ok, errs := conf.Check()
+			var err error
+			if len(errs) > 0 {
+				var s []string
+				for _, e := range errs {
+					s = append(s, e.Error())
+				}
+				err = fmt.Errorf("config file checker error: %v", strings.Join(s, ", "))
```

Comment:
I'm not sure how to format this error better at the moment,
I think this would read clearer if each error was printed to it's own line. I think multierr handles this nicely. Wdyt about using multierr here?